sample/android/app/src/main/java/io/sentry/sample/MainApplication.java

```diff
@@ -25,7 +25,19 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       @SuppressWarnings("UnnecessaryLocalVariable")
       List<ReactPackage> packages = new PackageList(this).getPackages();
-      packages.add(new RNSentryPackage());
+
+      boolean packageExists = false;
+      for (ReactPackage pkg : packages) {
+        if (pkg instanceof RNSentryPackage) {
+          packageExists = true;
+          break;
+        }
+      }
+
+      if (!packageExists) {
+        packages.add(new RNSentryPackage());
+      }
```

Comment:
why is this necessary?

Reply:
When the SDK is installed via linking to the parent folder for development we need to add the package manually with ` packages.add(new RNSentryPackage())`, however when the SDK is packaged and installed for the e2e tests React Native auto-adds the package already on Android, if the line is there then the app crashes at runtime for duplicate packages.